.github/workflows/new-locales.yml

```diff
@@ -0,0 +1,35 @@
+name: Add new locales
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - app/src/main/res/values-*/strings.xml
+permissions:
+  actions: none
+  checks: none
+  contents: write
+  deployments: none
+  discussions: none
+  id-token: none
+  issues: none
+  packages: none
+  pages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+jobs:
+  update-locales:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4.1.0
+    - name: Add new locales
+      run: .scripts/new-locales.py
```

Comment:
Wouldn't it make sense to merge this with the current `update-locales.yml` workflow? Otherwise we will get 1 MR to add the new locale and after merging another MR to add that same locale to `locales_config.xml`.

Reply:
I thought of that. But considered it might be useful not to. Keeping them split would allow making modifications to the PR this generates if we want to override something for some reason, before the other one runs (that one just syncs and should never need manual tweaking or more than cursory review).

Reply:
Forgot to mention that. Did intend to discuss that with you :sweat_smile:

Reply:
> I can modify MR content if needed, separating the scripts doesn't change that.

Yes. But it seemed easier to me if you don't need to mess with the other PR's sync and can modify the output of this one before the other one runs instead of having to modify the *combined* output. That was my personal preference. But it's up to you, so I'll combine the two actions into 1 workflow.

Reply:
> we can be certain we'll never have an inconsistent state no matter what.

That is of course a very valid argument :)